content/docs/ops/service-disruption-guide.md

```diff
@@ -62,3 +62,23 @@ A postmortem is not necessary to close, and it does not have to happen immediate
 The person who closes the event should then put a card on the component-owning squad’s board in the urgent lane about writing and posting a postmortem.
 
 For non-security-sensitive work, work to resolve root causes does not have to be completely done before we post the postmortem.
+
+### Checklist for drafting postmortems
+
+We typically use a **What happened** + **What we're doing** structure.
+
+Try checking your draft against the following questions that customers typically have:
+
+* What part of the system had a problem, in terms that I recognize?
+* How long did the problem last?
+* What was the effect on my applications and my own users?
```

Comment:
Can we highlight this as important? We need to clearly answer what user-facing application downtime was (if any).